ui/app/AppMain.qml

```diff
@@ -343,6 +343,7 @@ StatusAppLayout {
         property bool isBrowserEnabled: false
         property bool isActivityCenterEnabled: false
         property bool showOnlineUsers: false
+        property bool isGifEnabled: false
```

Comment:
Again, probably better to rename it `isGifSearchEnabled` or widget. Here it's for us devs, so I'd change it now

Reply:
Yeah I liked gif widget, I will change that